ui/app/adapters/config-ui/message.js

```diff
@@ -10,9 +10,9 @@ export default class MessageAdapter extends ApplicationAdapter {
     return 'config/ui/custom-messages';
   }
 
-  query(store, type, query) {
+  async query(store, type, query) {
```

Comment:
I'm curious why the `async...await` is necessary here? Query is expected to return a promise (just like the ajax requests below) and so I would think this is handled by the route model hook which already waits for all promises to resolve. But I've also not seen us use `super.query()` so I may be missing context

Reply:
Yup! You're right - I don't need `async...await` :) I forgot remove this!

Jordan made a suggestion to use `super.query()`! https://github.com/hashicorp/vault/pull/24407#discussion_r1423011063